src-literatetests/tests-context-free.blt

```diff
@@ -739,62 +749,70 @@ import qualified Data.List as L (foldl') {- Test -}
 
 #test import-with-comments-2
 
-import Test ( abc
-            , def
-            -- comment
-            )
+import Test
+  ( abc
+  , def
+  -- comment
+  )
 
 #test import-with-comments-3
 
-import Test ( abc
-            -- comment
-            )
+import Test
+  ( abc
+  -- comment
+  )
 
 #test import-with-comments-4
-import Test ( abc
-            -- comment
-            , def
-            , ghi
-            {- comment -}
-            , jkl
-            -- comment
-            )
+import Test
+  ( abc
+  -- comment
+  , def
+  , ghi
+  {- comment -}
+  , jkl
+  -- comment
+  )
 
 -- Test
 import Test (test)
 
 #test import-with-comments-5
-import Test ( -- comment
-            )
+import Test
+  ( -- comment
+  )
 
 #test long-bindings
 import Test (longbindingNameThatoverflowsColum)
 import Test (Long(List, Of, Things))
 
 #test things-with-with-comments
-import Test ( Thing( With
-                   -- Comments
-                   , and
-                   -- also
-                   , items
-                   -- !
-                   )
-            )
-import Test ( Thing( Item
-                   -- and Comment
-                   )
-            )
-import Test ( Thing( With
-                   -- Comments
-                   , and
-                   -- also
-                   , items
-                   -- !
-                   )
-            )
+import Test
+  ( Thing( With
+         -- Comments
```

Comment:
These are the only ones that look weird to me. The indentation depends on the length of the `Thing` identifier. I don't think I've ever actually had an import like this, but I think I'd format it as:

``` hs
import Test
  ( Thing
    ( With
    -- Comments
    , and
    -- also
    , items
    -- !
    )
  )
```

Reply:
Ah I should have expected this. I'll take a look.